cmake/modules/SwiftUtils.cmake

```diff
@@ -217,20 +217,14 @@ function(swift_create_early_driver_symlinks target)
       return()
   endif()
 
-  message(STATUS "Creating early SwiftDriver symlinks.")
+  message(STATUS "Copying over early SwiftDriver executable.")
   message(STATUS "From: ${driver_bin_dir}/swift-driver")
   message(STATUS "To: ${swift_bin_dir}/swift-driver")
-  swift_create_post_build_symlink(swift-frontend
-    SOURCE "${driver_bin_dir}/swift-driver"
-    DESTINATION "${swift_bin_dir}/swift-driver"
-    COMMENT "Creating early SwiftDriver symlinks: swift-driver")
+  configure_file(${driver_bin_dir}/swift-driver ${swift_bin_dir}/swift-driver COPYONLY)
```

Comment:
Hmm, I suspect that you are using `configure_file` rather than `file(COPY ...)` to create a dependency?  `file(COPY ...)` would be more natural IMO, otherwise this does deserve a comment.